src/scripts/main.js

```diff
@@ -1,3 +1,68 @@
 'use strict';
 
-// write code here
+// const header = document.querySelector('thead');
+// const bodyTable = document.querySelector('tbody');
+
+// header.addEventListener('click', (e) => {
+//   const index = e.target.cellIndex;
+//   const rows = bodyTable.rows;
+
+//   const sortedList = [...rows].sort((a, b) => {
+//     const row1 = a.cells[index].innerText.replace(/[$,]/g, '');
+//     const row2 = b.cells[index].innerText.replace(/[$,]/g, '');
+
+//     if (e.target.innerText === 'Name' || e.target.innerText === 'Position') {
+//       return row1.localeCompare(row2);
+//     } else {
+//       return row1 - row2;
+//     }
+//   });
+
+//   bodyTable.append(...sortedList);
+// });
+
+const header = document.querySelector('thead');
+const table = document.querySelector('tbody');
+
+function getNumber(string) {
+  return string.slice(1).split(',').join('');
+};
+
+header.addEventListener('click', (e) => {
+  const sortBy = e.target.innerText;
+
+  switch (sortBy) {
+    case 'Name':
+      const sortByName = [...table.rows].sort((a, b) => {
+        return a.cells[0].innerText.localeCompare(b.cells[0].innerText);
+      });
+
+      table.append(...sortByName);
+      break;
+
+    case 'Position':
+      const sortByPosition = [...table.rows].sort((a, b) => {
+        return a.cells[1].innerText.localeCompare(b.cells[1].innerText);
+      });
+
+      table.append(...sortByPosition);
+      break;
```

Comment:
you have similar logic for cases here - try to unite this logic, group cases and reuse logic